lib/palindrome_permutation.rb

```diff
@@ -1,5 +1,26 @@
 
 def palindrome_permutation?(string)
```

Comment:
You are very close here, just a few tweaks and you'd have it.  Take a look below.